contrib/examples/clusterapi-template.yml

```diff
@@ -141,16 +141,40 @@ objects:
     template:
       metadata:
         labels:
-          # TODO: enforce this in validation
           clusteroperator.openshift.io/machineset: ${CLUSTER_NAME}-master
           clusteroperator.openshift.io/cluster: ${CLUSTER_NAME}
       spec:
         providerConfig:
           value:
             apiVersion: clusteroperator.openshift.io/v1alpha1
+            # Manually populating, this will be set automatically based on our ClusterDeployment.
```

Comment:
I don't think that this comment is really necessary. These are things that a user would have to supply themselves if they were to create a MachineSet manually. The **entire** `MachineSet` would be populated manually instead of automatically by a `ClusterDeployment`.